src/diem/offchain/client.py

```diff
@@ -135,7 +135,21 @@ def send_request(
         return cmd_resp
 
     def process_inbound_request(self, request_sender_address: str, request_bytes: bytes) -> Command:
-        """Validate and decode the `request_bytes` into `diem.offchain.command.Command` object.
+        """Deprecated, validate and decode the `request_bytes` into `diem.offchain.command.Command` object.
```

Comment:
maybe move "validate and decode to a new line so Deprecated is even more emphasize 🤷